zerver/lib/cache.py

```diff
@@ -507,6 +507,10 @@ def realm_user_dicts_cache_key(realm_id: int) -> str:
     return f"realm_user_dicts:{realm_id}"
 
 
+def partial_realm_user_dicts_cache_key(realm_id: int) -> str:
+    return f"partial_realm_user_dicts:{realm_id}"
```

Comment:
I don't think we need to cache this in `memcached` -- it should be small enough as a data set that we can just query it from the database the normal way.